### DISCUSSION TITLE
Multi-format options

### DISCUSSION THREAD

**Discussion by msh855** [quarto-dev/quarto-cli#5354] [Q&A] [answered]
Is there a way to switch off the option for the table of contents for a specific format? For example, 

In my post, I have these options: 

```
format: 
  html: default
  pdf: default
```

And my global settings (i.e. yml file) I have as a default to have `toc` for all posts on my website. 

I am wondering, however, if I can keep the `toc` in the HTML and not in PDF. For example, I want my live post on my website to have the `toc`, but not for the pdf.

**Comment by mcanouil** [comment, marked as answer]
I'am away from a keyboard, but did you try to add `toc: false`?

**Reply by msh855** [reply]
Yes, it worked. I didn't know that works this way. Thanks

**Reply by msh855** [reply]
for reference to others here what worked for me: 

```
format: 
  html:
    code-fold: true
  pdf:
    toc: false
```

**Reply by mcanouil** [reply]
It's a format/project option as described in "reference" pages of the documentation.
Usually, it saves time to try and see it works or does not, before asking it it could work or not. 😉
